README.md

```diff
@@ -3,13 +3,19 @@
 ![](https://img.shields.io/github/v/release/hmrc/catalogue-frontend)
 
 * [Setup](#setup)
+* [Development](#development)
 * [Updating the front page](#updating-the-front-page)
 * [Tests](#tests)
 * [License](#license)
 
 ## Setup
 
-See [catalogue-acceptance-tests]("https://github.com/hmrc/catalogue-acceptance-tests") for help setting up dependent services.
+For help setting up dependent services see [catalogue-acceptance-tests]("https://github.com/hmrc/catalogue-acceptance-tests")
+
+## Development
+
+In order to run unit tests against any changes to views or partials you need to execute
```

Comment:
`sbt test` already invokes `compile` - did you mean `sbt clean`?

Reply:
Sorry, I should have clarified that comment - "run unit tests in the IDE" ... maybe there is an IDE setting but OOTB running a unit test (right click on it in the IDE) should run the required steps for converting partials to scala and then compiling that.  I couldn't see a task that did just the work of partial -> scala so the only solution seemed to be compile.  That seemed to work and appeared to be incremental anyway so not too time consuming.  Maybe there is a way to get the IDE to do this?  Or maybe the advice should be in the README so we don't have to get into IDE config or how people develop code